pages/index.tsx

```diff
@@ -1,24 +1,23 @@
-import type { GetStaticPaths, NextPage } from 'next'
-import PackageList from '../components/PackageList'
+import type { GetStaticPaths, NextPage } from "next";
+import PackageList from "../components/PackageList";
 import { listPackageNames } from "./api/services/PackageReader";
 
 const DUMMY_PACKAGE_LIST = [
   {
     name: "python-pkg-resources",
-    version: "0.6.24-1ubuntu1"
+    version: "0.6.24-1ubuntu1",
   },
   {
     name: "tcpd",
-    version: "1:2.4.0-5"
-  }
-]
-
-const Home: NextPage<{packageList: {name: string, version: string}[]}> = (props) => {
-  return (
-    <PackageList items={props.packageList}></PackageList>
-  )
-}
+    version: "1:2.4.0-5",
+  },
+];
 
+const Home: NextPage<{ packageList: { name: string; version: string }[] }> = (
+  props
+) => {
+  return <PackageList items={props.packageList}></PackageList>;
+};
```

Comment:
Here's an example where it might even make more sense (in a "tidier code"-sense) to not have an explciit function body

```typescript
const Home: NextPage<{ packageList: { name: string; version: string}[] } = 
  ({packageList}) => <PackageList items={packageList} />
```